<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-17T20:00:34 IST" name="Suite" finished-at="2023-07-17T20:00:42 IST" duration-ms="8864">
    <groups>
    </groups>
    <test started-at="2023-07-17T20:00:34 IST" name="Test" finished-at="2023-07-17T20:00:42 IST" duration-ms="8864">
      <class name="Clarion.Maven_Project.EndtoEnd">
        <test-method is-config="true" signature="initilizedriver()[pri:0, instance:Clarion.Maven_Project.EndtoEnd@7905a0b8]" started-at="2023-07-17T20:00:34 IST" name="initilizedriver" finished-at="2023-07-17T20:00:40 IST" duration-ms="5891" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initilizedriver -->
        <test-method signature="order(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Clarion.Maven_Project.EndtoEnd@7905a0b8]" started-at="2023-07-17T20:00:40 IST" name="order" data-provider="getdata" finished-at="2023-07-17T20:00:42 IST" duration-ms="2553" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Rahul@123.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[@123123abcA]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ZARA COAT 3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Lambda expressions are allowed only at source level 1.8 or above
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Lambda expressions are allowed only at source level 1.8 or above

at pageObjects.ProductCatlog.getbyproductname(ProductCatlog.java:42)
at pageObjects.ProductCatlog.Addprodtocart(ProductCatlog.java:52)
at Clarion.Maven_Project.EndtoEnd.order(EndtoEnd.java:61)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- order -->
        <test-method is-config="true" signature="close()[pri:0, instance:Clarion.Maven_Project.EndtoEnd@7905a0b8]" started-at="2023-07-17T20:00:42 IST" name="close" finished-at="2023-07-17T20:00:42 IST" duration-ms="129" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method signature="Orderhistorytest(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Clarion.Maven_Project.EndtoEnd@7905a0b8]" started-at="2023-07-17T20:00:42 IST" name="Orderhistorytest" finished-at="2023-07-17T20:00:42 IST" depends-on-methods="Clarion.Maven_Project.EndtoEnd.order" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Clarion.Maven_Project.EndtoEnd.Orderhistorytest() on instance Clarion.Maven_Project.EndtoEnd@7905a0b8 depends on not successfully finished methods 
[Clarion.Maven_Project.EndtoEnd.order() on instance Clarion.Maven_Project.EndtoEnd@7905a0b8]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Clarion.Maven_Project.EndtoEnd.Orderhistorytest() on instance Clarion.Maven_Project.EndtoEnd@7905a0b8 depends on not successfully finished methods 
[Clarion.Maven_Project.EndtoEnd.order() on instance Clarion.Maven_Project.EndtoEnd@7905a0b8]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Orderhistorytest -->
      </class> <!-- Clarion.Maven_Project.EndtoEnd -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
